tests/cypress/tests/e2e/mcpar/dashboard.stepdef.js

```diff
@@ -0,0 +1,22 @@
+import { Then } from "@badeball/cypress-cucumber-preprocessor";
+
+Then("there is the active program with {string}", (category) => {
+  cy.contains(category, { matchCase: true }).should("be.visible");
+});
+
+Then("there is no way to create a program", () => {
+  cy.contains("Add managed care program").should("not.exist");
+});
+
+Then("the program is archived", () => {
+  cy.contains("Unarchive").should("be.visible");
+});
+
+Then("one program is archived and the other is unarchived", () => {
+  cy.findAllByRole("button", { name: "Archive" }).first().should("be.visible");
+  cy.findAllByRole("button", { name: "Unarchive" }).last().should("be.visible");
+});
```

Comment:
Similar here: can we count on this if we know other tests and we as manual testers might create programs?